Libplanet/Blocks/InvalidBlockTotalDifficultyException.cs

```diff
@@ -0,0 +1,23 @@
+using System;
```

Comment:
We turn on `nullable` check for new added files.

```suggestion
#nullable enable
using System;
```